crates/apub/src/objects/community.rs

```diff
@@ -138,11 +149,21 @@ impl ApubObject for ApubCommunity {
     context: &LemmyContext,
     request_counter: &mut i32,
   ) -> Result<ApubCommunity, LemmyError> {
-    let form = Group::into_form(group.clone());
+    let domain = generate_domain_url(group.id.inner())?;
+    let instance_form = InstanceForm {
+      domain,
+      updated: Some(naive_now()),
+    };
+    let instance = blocking(context.pool(), move |conn| {
+      Instance::create(conn, &instance_form)
+    })
+    .await??;
```

Comment:
I think this entire code block should be in a helper function, no need to copy-paste it everywhere.